src/components/Lend/LoanSearch/LoanSearchLocationFilter.vue

```diff
@@ -88,7 +88,19 @@ export default {
 		updateRegion(region, { values, changed, wasSelectAll }) {
 			this.$set(this.selectedCountries, region, values);
 
-			this.$kvTrackEvent?.('Lending', 'click-location-filter', wasSelectAll ? region : changed);
+			let appliedState = '';
+			if (wasSelectAll) {
+				appliedState = values.length ? 'select-all' : 'deselect-all';
+			} else {
+				appliedState = values.includes(changed) ? 'selected' : 'deselected';
+			}
+
+			this.$kvTrackEvent?.(
+				'Lending',
+				'click-location-filter',
+				wasSelectAll ? `Region: ${region}` : `Country: ${changed}`,
```

Comment:
I've "enhanced" these values to notate if the click was on a region vs a country. There is also a slight difference in the `appliedState`.